WordPress/Classes/ViewRelated/NUX/SigninHelpers.swift

```diff
@@ -342,31 +351,52 @@ import WordPressShared
     }
 
 
-    // MARK: - Helpers for Saved Magic Link Email
-
+    // MARK: - Helpers for Saved Magic Link Info
 
-    /// Saves the specified email address in NSUserDefaults
+    /// Saves certain login information in NSUserDefaults
     ///
-    /// - Parameter email: The email address to save.
+    /// - Parameter loginFields: The loginFields instance from which to save.
     ///
-    @objc class func saveEmailAddressForTokenAuth(_ email: String) {
-        UserDefaults.standard.set(email, forKey: AuthenticationEmailKey)
+    class func storeLoginInfoForTokenAuth(_ loginFields: LoginFields) {
+        let dict: NSMutableDictionary = [
```

Comment:
(No need to change anything!!). You can also use a Swift native type here:

```
var dict: [String: String] = [ ... ]
```

(And below something like `dict[Constants.jetpackBlogIDURL] = url`).